chef_master/source/audit_cookbook.rst

```diff
@@ -0,0 +1,264 @@
+Sending Compliance Data to Chef Automate with ``audit`` Cookbook
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/audit_cookbook.rst>`__
+
+
+.. note:: The ``audit`` cookbook uses the InSpec gem. It will check for an installed version of InSpec and install it upon finding none. The latest version of InSpec will be installed by default unless otherwise specified with ``node['audit']['inspec_version']``.
+
+:doc:`Cookbooks </cookbooks>` are Chef's primary unit of configuration management.  For tutorials on working with cookbooks in Chef, see `Learn Chef Rally <https://learn.chef.io>`_.
+For more information specifically on using the ``audit`` cookbook with Automate, see :doc:`perform a compliance scan </perform_compliance_scan>`.
+
+The ``Audit`` Cookbook
+=================================
+.. tag audit_cookbook_420
+
+.. note:: Audit Cookbook version 4.2.0 or later requires InSpec 1.25.1 or later. You can upgrade your InSpec package in several different ways: by upgrading Automate, by upgrading the Chef Development Kit, by upgrading Chef Client, or by setting the ``node['audit']['inspec_version']`` attribute in your cookbook.
+
+.. end_tag
+
+To send compliance data gathered by InSpec as part of a Chef client run, you will need to use the `audit cookbook <https://github.com/chef-cookbooks/audit>`_. All profiles configured to run during the audit cookbook execution will send their results back to the Chef Automate server.
+
+Configure the Node for ``audit`` Cookbook
+---------------------------------------------
+Once the cookbook is available in Chef Server, you will need to add the recipe to the run-list of each node. Compliance profiles are added using the ``node['audit']['profiles']`` attribute. A complete list of the possible configuration are documented in :doc:`Audit Cookbook </audit_cookbook>`.
+
+To configure the audit cookbook to report compliance data directly to Chef Automate, you will first need to configure the Chef client to send node converge data, as described in :doc:`Data Collection </data_collection>`. Once you have done that, configure the audit cookbook's collector by setting the ``collector``, ``server``, ``owner``, ``refresh_token`` and ``profiles`` attributes.
+
+   * ``reporter`` - ``'chef-automate'`` to report to Chef Automate.
+   * ``server`` - url of Chef Automate server with ``/api``.
+   * ``owner`` - Chef Automate user or organization that will receive this scan report.
+   * ``refresh_token`` - refresh token for Chef Automate API. Please note that logging out of the user interface revokes the refresh_token. To workaround, log in once in a private browser session, grab the token and then close the browser without logging out.
```

Comment:
When the refresh token is mentioned in the second sentence, it's referring to the token as a concept - so I don't think there's any need for the underscore that is present in the config file. If it's meant to reference the `refresh_token` setting the config file, i think it should be in `` since it's referring to the setting as it appears in a file.

Reply:
done